### PR TITLE
Add chevron after See more to the list header

### PR DESCRIPTION
**Pull request by bor0**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/woocommerce.com/issues/21573.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check current site language `wp option get WPLANG` - should be `en_US`
2. Go to http://localhost:8080/wp-admin/admin.php?page=wc-admin&path=%2Fextensions
3. Make sure there's a chevron after `See more`

Screenshot 2024-09-06 at 15 23 21

4. Install a RTL language `wp language core install ar`
5. Switch to a RTL language `wp language core activate ar`
6. Go to http://localhost:8080/wp-admin/admin.php?page=wc-admin&path=%2Fextensions
7. Make sure there's a chevron after `See more` and it renders properly

Screenshot 2024-09-06 at 15 23 03

8. Switch back to English `wp language core activate en_US`

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.tsx [hunk 1]
```diff
@@ -2,9 +2,10 @@
  * External dependencies
  */
 import { Link } from '@woocommerce/components';
-import { __ } from '@wordpress/i18n';
+import { isRTL, __ } from '@wordpress/i18n';
 import clsx from 'clsx';
 import { recordEvent } from '@woocommerce/tracks';
+import { Icon, chevronLeft, chevronRight } from '@wordpress/icons';
```
Comment (andfinally) [inline plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.tsx]: I like to list imports in alphabetical order, but I suppose I'm just being pedantic.

**Review by andfinally — APPROVED** [review]
Lovely! Kudos for remembering RTL languages! 👌

I was wondering about the colour of the chevron – the issue assumes it'll be coloured same as the link.

<p>
image
</p>

We could colour it like this.

```diff
Index: plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.scss b/plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.scss
--- a/plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.scss	(revision 467bd812a8bb8f08b38c1fad1325809e4170f3e4)
+++ b/plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.scss	(date 1725981541929)
@@ -41,5 +41,9 @@
 		justify-content: center;
 		align-items: center;
 		text-decoration: none;
+
+		path {
+			fill: #2271b1;
+		}
 	}
 }

```

<p>
image
</p>

Unfortunately, that `#2271b1` colour on the link comes from WordPress `wp-admin/css/common.css`. Most people will accept that default, but some might override it with an admin theme. If so, our chevron might look weird. I'm in two minds whether it's better to leave it as black for everyone, or make it `#2271b1` for most people.

The chevron also doesn't look perfectly vertically aligned with the text to me, despite being properly centred. But I've not found any way to improve that that isn't a nasty hack. We'll probably just have to accept it as one of those peculiarities of browser text rendering.

File: plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.tsx (hunk 1 above)
Comment (bor0) [reply plugins/woocommerce-admin/client/marketplace/components/product-list-header/product-list-header.tsx]: Your pedantry satisfied in a991fa2c9c :)

**Comment by bor0** [comment]
@andfinally thanks for the in-depth review! Happy to discuss those further and potentially tackle them as part of separate issue - I see we use chevrons in a bunch of different places.

**Pull request merged**